third_party/algorand/requirements.txt

```diff
@@ -0,0 +1,5 @@
+pyteal==0.9.0
+py-algorand-sdk==1.8.0
+mypy==0.910
+pytest
```

Comment:
We should pin this dependency to make this build deterministic

Reply:
How does one do this?

Reply:
Running `pip freeze -l` should list the version you're using

Reply:
Take a look now